### PR TITLE
identifying a payment system

### PR DESCRIPTION
**Pull request by trayhorn**
Bank Card component will be able to identify what is a payment system of a card by its first digit.

### CODE REVIEW

File: src/components/CardsSlider.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import BankCard from './BankCard'
 import { Card } from '../typescript/types'
```
Comment (maksimuchhh) [inline src/components/CardsSlider.tsx:4]: Remove "type" key from this type. 
Also remove type from cardsList on Home page

File: src/components/BankCard.tsx [hunk 2]
```diff
@@ -9,6 +9,11 @@ type Props = {
 }
 
 function BankCard({ number, currency, amount, type }: Props) {
```
Comment (maksimuchhh) [inline src/components/BankCard.tsx:11]: remove type

File: src/components/BankCard.tsx [hunk 3]
```diff
@@ -9,6 +9,11 @@ type Props = {
...
+  if (number?.charAt(0) === '4') {
```
Comment (maksimuchhh) [inline src/components/BankCard.tsx:12]: Remove if else construction and make a variable "type"
`const type = number?.charAt(0) === '4' ? 'Visa' : number?.charAt(0) === '5' ? 'MasterCard' : undefined`

File: src/components/BankCard.tsx [hunk 4]
```diff
@@ -21,6 +26,10 @@ function BankCard({ number, currency, amount, type }: Props) {
         </p>
       )}
       {type && <p className="bank-card__type">{type}</p>}
+
```
Comment (maksimuchhh) [inline src/components/BankCard.tsx:29]: remove this code

**Comment by trayhorn** [comment]
all done

**Pull request merged**